src/BootstrapInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+declare(strict_types=1);
+
+namespace LotGD\Core;
+
+use Symfony\Component\Console\Application;
+
+interface BootstrapInterface
```

Comment:
Let's talk about why we need this class instead of using values in `composer.json`.

Why not continue to read the entity paths from the `extra[lotgd-namespace]` like my code does?

Why not add commands via an `extra` field as well, something like `extra[lotgd-daenerys-commands]` which is an array of BaseCommand subclasses?

Using `composer.json` feels cleaner than code to me b/c it's declarative and can be validated. I don't think these things will ever be "computed" but will rather be a static path and list of commands.

Correct me if I'm wrong here.